src/args.ts

```diff
@@ -22,7 +23,11 @@ const handleNoCommand = async () => {
     },
   ];
 
-  const answers = await inquirer.default.prompt(questions);
+  //TODO: this is very sus - review this
+  const answers = (await inquirer.default.prompt(questions)) as {
+    command: string;
+  };
```

Comment:
[Note] I could not work out what type this object was meant to be. It would be great to chat to someone about what this package is doing as I don't feel like it is working nicely with typescript 🤔

Reply:
Inquirer was made in js. I have never used types with it but maybe someone has built some.

Reply:
Maybe I'll try and look into it as part of the casting ticket